.github/workflows/build.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - develop
+      - fix/1324-integration-tests
```

Comment:
Can we remove this?

Reply:
Niki is testing with this branch. We can remove it yes.

Reply:
removed